proxy/proxy.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/streadway/amqp"
+
+    "github.com/mishas/prometheus_amqp_proxy/proxy/config"
+    "github.com/mishas/prometheus_amqp_proxy/proxy/rpc"
+)
+
+type externalAuth struct{}
+
+func (a *externalAuth) Mechanism() string {
+	return "EXTERNAL"
+}
+func (a *externalAuth) Response() string {
+	return fmt.Sprintf("\000")
+}
+
+// getAMQPConfig returns a reference to the amqp.Config object.
+// certsDir should be the path to the directory holding {cacert,cert,key}.pem files for the TLS
```

Comment:
Fix the comment to indicate that you now use file config variables instead of a directory.